### DISCUSSION TITLE
ACTUALIZAR GIT

### DISCUSSION THREAD

**Discussion by ismagician** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#587] [General]
<h3> Hola compañeros Explorers 🚀 </h3>
 
El día de ayer se dieron a conocer dos vulnerabilidades de Git de alto riesgo. 
La vulnerabilidad CVE-2022-24765 afecta a equipos multi-usuario donde se tengan carpetas compartidas; con esta vulnerabilidad se  puede llegar a permitir la ejecución de comandos.
Mientras que la vulnerabilidad CVE-2022-24767 afecta  a usuarios de Windows ya que al ejecutar el desisntalador de Git se pueden implantar archivos ``` .dll ``` maliciosos   
Se recomienda actualizar a la versión **2.35.2 de Git** para protegerse de estas vulnerabilidades. 
Les dejo una pequeña guía para actualizar Git en Windows:
Para saber que versión tienen con el comando ``` git --version```

![imagen](https://user-images.githubusercontent.com/92187317/163280243-3174d920-8cef-4d98-a79a-71c6be17254b.png)

Para actualizar, desde cmd o la consola de git usen el comando ``` git update-git-for-windows ```

![imagen](https://user-images.githubusercontent.com/92187317/163281008-1176e944-4614-4c05-88e7-a498713c8ea4.png)

Les abre el instalador, aceptan y les sale lo siguiente, esperan a que se instale. 
![imagen](https://user-images.githubusercontent.com/92187317/163281161-d8ea4848-5d24-4522-9210-a62cdb097190.png)

Cuando termine comprobar la versión Git
![imagen](https://user-images.githubusercontent.com/92187317/163285993-7f3ecbb5-546f-48c8-8f76-38e140e6a89a.png)

Les dejó el blog GitHub donde explican estas vulnerabilidades 
https://github.blog/2022-04-12-git-security-vulnerability-announced/
Y la página donde encontré la noticia
https://portswigger.net/daily-swig

**Comment by CarlosCastilloHdz** [comment]
Muchas gracias mi estimado, Saludos
